frontend/public/components/monitoring/targets.tsx

```diff
@@ -249,11 +254,11 @@ const tableClasses = [
   'pf-m-hidden pf-m-visible-on-md', // Scrape Duration
 ];
 
-const Row: React.FC<RowFunctionArgs<Target>> = ({ obj }) => {
-  const { health, labels, lastScrape, lastScrapeDuration, scrapeUrl } = obj;
+const Row: React.FC<RowProps<Target>> = ({ obj }) => {
+  const { health, labels, lastError, lastScrape, lastScrapeDuration, scrapeUrl } = obj;
 
-  const isServiceMonitor: boolean = obj.scrapePool.includes(MonitorType.ServiceMonitor);
-  const isPodMonitor: boolean = obj.scrapePool.includes(MonitorType.PodMonitor);
+  const isServiceMonitor: boolean = obj && obj.scrapePool.includes(MonitorType.ServiceMonitor);
```

Comment:
`obj` is always truthy as it is destructured above

Reply:
👍 
Changed it to include `scrapePool` in the destructuring above.